i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/notes.mdx

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_label: Notizen
+title: Notizen
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Notizen</h2>
+    <p class="hero__subtitle">Speichern Sie wichtige Nachrichten, Medien und sogar Nachrichten an sich selbst!</p>
```

Comment:
Please change this to: "Speichern Sie wichtige Nachrichten, Medien und schreiben Sie sich sogar selbst Nachrichten!"